lite/api/benchmark.cc

```diff
@@ -61,10 +64,7 @@ DEFINE_int32(threads, 1, "threads num");
 DEFINE_string(result_filename,
               "result.txt",
               "save the inference time to the file.");
-DEFINE_bool(run_model_optimize,
-            false,
-            "if set true, apply model_optimize_tool to "
-            "model and use optimized model to test. ");
+DEFINE_bool(is_show_output, false, "Wether show the output.");
```

Comment:
is_show_output -> show_output ?   should add more comments.

Reply:
Done